wiki/Ranking_Criteria/zh.md

```diff
@@ -1,300 +1,303 @@
-- **命令（Command）：** 它们以各种方式改变精灵图的位置和大小。命令包含 `Move`（移动），`Scale`（缩放），`Fade`（隐藏）和 `Rotate`（旋转）等等。每一条命令都有开始时间和结束时间。
-- **坐标轴命令（Axis Specific Command）：** 只会影响特定空间坐标轴的命令，例如 `MoveX` 和 `MoveY`。
-- **激活时间（Active）：** 在对象的命令中，从开始到最后结束之间的时间。
-- **正被渲染的（Rendered）：** 通常指尚未在屏幕中完全消失的精灵图。
-- **osu! 像素（osu!pixel）：** 设计标签页中的最小尺寸。可以在编辑器屏幕的右上角看到，例如 `x: 104; y: 88`。
+- **故事板图片 (Storyboard Image)：** 指故事板使用的图片，它们都存放在谱面文件夹中。
+- **精灵图 (Sprite)：** 一个表示故事板中一张图片或一组图片的对象。
+- **时间 (Time)：** 表示某一对象在时间轴上的位置，以毫秒为单位。这通常能在谱面编辑器的设计页看到。
+- **命令 (Command)：** 游戏有 `Move`（移动）、`Scale`（缩放）、`Fade`（淡化）、`Rotate`（旋转）等命令。这些命令能操纵并改变精灵图的位置、大小、朝向、透明度等。每条命令都有开始时间和结束时间。
+- **坐标轴命令 (Axis Specific Command)：** 指仅仅在特定坐标轴方向上影响精灵图的命令，例如 `MoveX` （X 轴方向移动）和 `MoveY`。（Y 轴方向移动）
+- **激活时间 (Active)：** 指控制对象的命令从开始到结束这段时间。
+- **正在渲染 (Rendered)：** 通常用来描述在屏幕中，且未完全淡化成透明的精灵图。
+- **osu! 像素 (osu!pixel)：** 谱面编辑器设计页中的最小尺寸。可以在设计页右上角看到它，例如 `x: 104; y: 88`。
 
 ### 规定
 
-- **为了保证大部分电脑都能在短时间内加载完成，故事板图像所包含的总像素数量不能超过一千七百万。** 此外，你需要根据故事板编辑器内部最大尺寸（854 x 480 像素）来缩放和使用你的图像。
-- **谱面在加载时，不能引发解析错误。** 这意味着解析器无法读取某一部分故事板指令。
-- **`Widescreen support`（宽屏支持）设置必须在谱面内保持一致。** 除非各难度的故事板是为不同宽高比设计的。
-
-### 准则
-
-- **在歌曲结束后不应有仍被激活的精灵图和命令。** 为了某些故事板的特定效果需要，精灵图的渲染时长可以延长到歌曲结束后一段时间，但不应过长。
-- **考虑在用于旋转的精灵图的故事板图片周围绘制 1px 的透明边框。** osu! 没有图像的抗锯齿算法，因此，如果边缘清晰的精灵图用作旋转，锯齿或边框将会非常明显。
-- **尽量避免任何明显的性能问题。即使优化过，保持帧速率一致对于打图体验也是至关重要的。** 请在摸图时测试谱面来确认这一点。
-- **避免使用故事板音效，在打图过程中容易与敲击音效混淆。** 这违背了听觉反馈这一概念，因为故事板音效将独立于玩家的操作而自动播放。
-- **避免使用不合逻辑、相互冲突和已经作废的命令。** 游戏无法正常执行间隔重叠、结束时间早于开始时间或无法成功触发的这类命令。应删除或修正。
-- **如果谱面中包含宽屏故事板，则应启用 `Widescreen support`（宽屏支持）。** 反之，如果故事板的大小设计为 4:3 分辨率，则应关闭此功能。如果没有故事板，此设置不会影响谱面中的任何内容。
-- **请尽可能优化故事板，** 无论如何都要做到这一点。
-  - **避免精灵图或谱面背景在渲染时被完全遮挡至不可见。** 出于性能考虑，当元素不可见时，最好将它们隐藏。如要隐藏谱面的背景，请将相同的背景图像作为精灵图，`Background` 或 `0` 作为第二个参数，然后隐藏掉它。
-  - **避免精灵图在整个渲染过程中离开屏幕或者阻碍视线。** 在这种情况下，应该裁剪图像的不可见部分，除非它们对故事板中的效果来说是必要的。
-  - **避免故事板图像周围不必要的透明。** 为了提高性能，应在保留所需效果下尽可能多地裁剪图像。
-  - **若要多次使用某条命令，请使用循环指令，除非这样做会影响视觉效果。** 使用 Loop 指令通常会大量减少代码行数，从而减少文件大小。
-  - **当使用一个通用命令可以实现相同的效果时，请避免使用两个坐标轴分量命令。** 使用单条命令而不是两条命令，能减少文件大小。
-  - **使用最节省空间的图像文件格式，同时保持合理的质量。** 不同于 `.jpg` 格式，`.png` 格式由于无损压缩方式，文件通常更大。
-  - **避免任何重复的图像文件。** 完全相同内容的两个图像会增加不必要的文件大小。
-  - **避免同时激活多个不可见的精灵图。** 无论元素是否可见，游戏仍然会处理元素的命令。如果这种元素不可见情况的持续时间较长，请创建一个新的元素在其重新出现时启用。
-  - **在同一个精灵图上使用相同类型的多个指令时，请使它们的激活时间间隔至少保留 16 毫秒。** 每秒 60 个命令足以使精灵图视觉上平滑过渡，且可以减小文件大小，减少加载时间。
-  - **由触发器激活的精灵图要在使用后隐藏。** 触发器会从第一个激活命令启用，保持激活状态直到谱面结束，所以最好在不使用后隐藏它。
+- **故事板图片所包含的总像素数量不能超过 1700 万，这是为了保证大部分电脑能在短时间内加载完成。** 此外，你需要根据故事板编辑器内部最大尺寸（854 x 480 osu! 像素）来缩放图片。
+- **谱面在加载时，不能引发解析错误。** 即解析器无法读取故事板指令的某一部分。
+- **谱面的`宽屏支持` (Widescreen Support) 设置必须保持一致。** 但如果不同难度的故事板是为不同宽高比设计，则不受这一规定的限制。
+
+### 指南
+
+- **在歌曲结束后，不应存有激活精灵图和命令。** 为了某些故事板的特定效果需要，精灵图的渲染时长可以延长到歌曲结束后一段时间，但不应过长。
+- **在含有旋转指令的故事板图片周围绘制 1 像素宽的透明边框。** osu! 没有图片的抗锯齿算法。因此，如果用作旋转的故事板图片边缘非常锐利，旋转中的锯齿将会非常明显。
+- **尽量保持故事板播放时帧率稳定，尽量避免任何明显的性能问题。** 请在摸图时测试谱面来确认这一点。
+- **避免使用在游玩过程中容易与打击音效混淆的故事板音效。** 这违背了听觉反馈这一概念，因为故事板音效将独立于玩家的操作而自动播放。
+- **避免使用不合逻辑、相互冲突或已经作废的命令。** 游戏无法正常执行激活时间重叠、结束时间早于开始时间等无法成功触发的命令。应删除或修正它们。
```

Comment:
> **Avoid illogical, conflicting and obsolete commands. **Commands that have their ending time before their start time or are bound to impossible to reach triggers are either not working as intended or obsolete, and should either be removed or adjusted to work as intended. Commands of the same type whose intervals overlap should have their intervals and parameters adjusted so that they no longer overlap.

原文这么长，怎么这里就一句了（）

```suggestion
- **避免使用不合逻辑、相互冲突或已经作废的命令。** 截止时间早于开始时间的指令无法被激活或按照预期正常工作，已被废弃。应删除或调整他们以保证游戏正常工作 。而区间重叠的指令应对参数或区间进行调整以保证他们不再重叠。
```

应用直译的一个原因是重叠的指令并没有提到不能被工作，只是规定上不允许使用重叠的指令。

Reply:
你给的翻译版本太长了，我修改一下语序

Reply:
其实大致是一个意思就是了